test/system/reports_test.rb

```diff
@@ -4,100 +4,104 @@
 
 class ReportsTest < ApplicationSystemTestCase
   def setup
-    @user = FactoryBot.create(:user)
+    @user = FactoryBot.create(:user, name: 'ケン')
     login_as(@user.email, @user.password)
   end
 
   test 'elements on report index page' do
-    report = FactoryBot.create(:report, user_id: @user.id, created_at: Time.zone.parse('2023-08-30 12:00:00'))
-
-    visit reports_url
-    assert_selector 'h1', text: '日報の一覧'
-    within 'div.index-item' do
-      within "div#report_#{report.id}" do
-        assert_selector 'p', text: report.title
-        assert_selector 'p', text: report.content
-        assert_selector 'p', text: report.user.name
-        assert_selector 'p', text: '2023/08/30'
+    travel_to Time.zone.local(2023, 8, 30) do
+      report = FactoryBot.create(:report, user_id: @user.id, title: 'キウイ本を読んだ', content: 'ちょうど良い難易度でした')
+
+      visit reports_url
+      assert_text '日報の一覧'
+      within 'div.index-item' do
+        within "div#report_#{report.id}" do
```

Comment:
```suggestion
      within "#report_#{report.id}" do
```

IDはHTML内に1つしかないはずなので、これだけでOKです

Reply:
確かにっすね...